AmplifyTestCommon/Models/schema.graphql

```diff
@@ -15,3 +16,9 @@ type Comment @model {
     createdAt: AWSDateTime!
     post: Post @connection(name: "PostComment")
 }
+
+enum PostStatus {
+    deleted
```

Comment:
Since this is updated, will have to update the provisioned backend for the API test as well

Reply:
i plan on pulling this PR locally and updating the API integration tests which will use the new Post model with an updated provisioned backend, so we can push the changes in one go without having to follow up on another PR. i think this will be a good test to ensure that this case functions in with a real backend